projects/storefrontapp-e2e-cypress/cypress.json

```diff
@@ -2,15 +2,16 @@
   "defaultCommandTimeout": 10000,
   "baseUrl": "http://localhost:4200",
   "requestTimeout": 15000,
+  "chromeWebSecurity": false,
   "retries": {
     "runMode": 2
   },
   "videoUploadOnPasses": false,
   "env": {
     "CLIENT_ID": "mobile_android",
     "CLIENT_SECRET": "secret",
-    "API_URL": "https://40.76.109.9:9002",
-    "BASE_SITE": "electronics-spa",
+    "API_URL": "https://api.cpce-teamtiger1-d1-public.model-t.cc.commerce.ondemand.com/",
+    "BASE_SITE": "powertools-spa",
```

Comment:
Let's move it to `cypress.cpq.json`. It can use "extends" property to extend from the main cypress.json (to avoid duplicate parts of config). See https://www.cypress.io/blog/2020/06/18/extending-the-cypress-config-file/